modules/ELT/pages/getting-started-with-airbyte.adoc

```diff
@@ -0,0 +1,197 @@
+= Airbyte with Teradata Vantage
```

Comment:
Let's change the title so that it's clear to the reader what it's about even if they don't know what AirByte is, e.g.:

```suggestion
= Use Airbyte to load data from external sources to Teradata Vantage
```